pkg/controller/che/che_controller.go

```diff
@@ -1123,3 +1089,51 @@ func isTrustedBundleConfigMap(mgr manager.Manager, obj handler.MapObject) (bool,
 		},
 	}
 }
+
+func (r *ReconcileChe) autoEnableOAuth(cr *orgv1.CheCluster, request reconcile.Request, isOpenShift4 bool) (reconcile.Result, error) {
+	var message, reason string; var oauth bool
```

Comment:
```suggestion
	var message, reason string
	oauth := false
```

Reply:
`var oauth bool` is the same like `oauth := false`. But ok I agree you proposed more readable variant.